enterprise-suite/scripts/lbc.py

```diff
@@ -276,10 +301,32 @@ def check_new_install_script():
         printout("info: New installer is available. Use the following command to download it: curl -O {}".format(installer_url))
 
 
+def helm_migration_check():
+    if helm_version > 2:
+        cmd = ('kubectl get configmap --namespace {} --selector OWNER=TILLER,NAME={} '
+               '--ignore-not-found=true'.format(args.tiller_namespace, args.helm_name))
+        printout(cmd)
+        returncode, stdout, _ = run(cmd)
+        if returncode != 0:
+            fail('\nwarning: Failed to check for legacy helm 2 installations.')
+        elif stdout != '':
+            message = (
+                '\nerror: Detected a previous Console installation which was installed with helm 2 '
+                'which has not been migrated to helm 3.\n'
+                'You must migrate these manually by following the instructions on '
```

Comment:
I find this wording a bit confusing.  Perhaps something along the lines of? :

Detected a previous Console installation which was installed with helm 2.  To use helm `helm_version` you must migrate the installation manually following the instructions on [...].